tests/foreman/cli/test_contentview.py

```diff
@@ -2647,6 +2647,7 @@ def test_positive_auto_update_composite_to_latest_cv_version(self):
             components[0]['version-id'], '{0} (Latest)'.format(version_2_id))
         self.assertEqual(components[0]['current-version'], '2.0')
 
+    @upgrade
```

Comment:
shouldn't the following tests cover this scenario?
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_contentview.py#L2737
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_contentview.py#L2888

Reply:
:+1: